### ISSUE TITLE
Is There A Way To Add Filters To The Entered Giveaways?

### ISSUE THREAD

**Issue by AshMan420** [TobiasPankner/Gleam-giveaway-bot#6]
I was running the script and saw this `Total giveaways after filtering: 646`. So this led me to think if I could enter the ones for the products I want.

**Comment by joernKoenen** [comment]
No, the bot saves which giveaways it already joined or it couldn't join because of an error. When you restart the bot it fetches all available giveaways and then checks each one if it already joined it and if so it filters it out. That way the bot only joins not already done giveaways each time.
